src/main/java/net/pms/util/CustomImageReader.java

```diff
@@ -0,0 +1,147 @@
+package net.pms.util;
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Iterator;
+import java.util.Locale;
+import javax.imageio.IIOException;
+import javax.imageio.ImageIO;
+import javax.imageio.ImageReadParam;
+import javax.imageio.ImageReader;
+import javax.imageio.spi.IIORegistry;
+import javax.imageio.spi.ImageInputStreamSpi;
+import javax.imageio.stream.ImageInputStream;
+import net.pms.util.ImagesUtil.ImageFormat;
+
+/**
+ * This is a hack of a utility class created because whoever wrote {@link ImageIO}
+ * decided to make it {@code final}. The sole purpose of these methods is to
+ * return the image format of the source image along with the {@link BufferedImage}.
+ *
+ * There should be no reason to have to do this twice, and thus the result from
+ * the analysis done when reading the image is kept.
+ */
+public class CustomImageReader {
+
+    protected static final IIORegistry theRegistry = IIORegistry.getDefaultInstance();
+
+	// Not to be instantiated
+	private CustomImageReader() {
+	}
+
+	/**
+	 * A copy of {@link ImageIO#read(InputStream)} that calls {{@link #read(ImageInputStream)}
+	 * instead of {@link ImageIO#read(ImageInputStream)}.
+	 *
+	 * @see ImageIO#read(InputStream)
+	 */
+    public static ImageReaderResult read(InputStream input) throws IOException {
+        if (input == null) {
+            throw new IllegalArgumentException("input == null!");
+        }
+
+        ImageInputStream stream = createImageInputStream(input);
+        ImageReaderResult result = read(stream);
+        if (result.bufferedImage == null) {
```

Comment:
@Sami32 I can't get that image to work nomatter what I do. Is the format valid? I can see that Windows can render it, but that doesn't mean it's valid. Does it have some unusual characteristic that we need to try to detect?

I've updated Twelvemonkeys' ImageIO plugins to the latest version (3.3.1) and it doesn't help. ImageIO manages to read it into a BufferedImage, but that must be corrupted somehow because trying to write that BufferedImage to either a TIFF or a JPEG fails.

I've made some corrections (not pushed yet) to handle it a little bit better, but there's not much I can do as the errors ImageIO throws aren't really usedful (```ArrayIndexOutOfBoundsException``` when trying to write TIFF and "```IIOException```: Invalid argument to native writeImage" when trying to write JPEG).

Given that the image is valid, you could try to post an issue with [TwelveMonkeys](https://github.com/haraldk/TwelveMonkeys/issues).